_assets/ci/Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
 
   options {
     /* Prevent Jenkins jobs from running forever */
-    timeout(time: 30, unit: 'MINUTES')
+    timeout(time: 40, unit: 'MINUTES')
```

Comment:
No specific objection to this, but I feel that we should probably think about optimising our tests so that we don't keep increasing the timeout ad infinitum. I think it used to be 5 minutes 😆